src/App.js

```diff
@@ -1,29 +1,42 @@
-import logo from './logo.svg';
+import {
+  createBrowserRouter,
+  RouterProvider,
+  Route,
+  createRoutesFromElements,
+} from 'react-router-dom';
+// import { ToastContainer } from 'react-toastify';
+import Reserve from './components/Reserve';
+import Doctors from './components/Doctors';
+import Doctor from './components/Doctor';
 import './App.css';
+import SignUp from './components/auth_pages/Signup';
+import Login from './components/auth_pages/Login';
+import MyReservations from './components/MyReservations';
+import AddDoctor from './components/AddDoctor';
+import DeleteDoctor from './components/DeleteDoctor';
+import Root from './components/Root';
+import ReservationForm from './components/ReservationForm';
+
+const router = createBrowserRouter(
+  createRoutesFromElements(
+    <Route>
+      <Route path="/" element={<Root />}>
+        <Route path="/" element={<Doctors />} />
+        <Route path="/add" element={<AddDoctor />} />
+        <Route path="/doctors/:doctorId" element={<Doctor />} />
+        <Route path="/delete" element={<DeleteDoctor />} />
+        <Route path="/reserve" element={<Reserve />} />
+        <Route path="/reservation" element={<ReservationForm />} />
+        <Route path="/reservations" element={<MyReservations />} />
+      </Route>
+      <Route path="/log_in" element={<Login />} />
+      <Route path="/sign_up" element={<SignUp />} />
+    </Route>,
+  ),
+);
 
 function App() {
```

Comment:
- [ ]  Please, kindly refactor this and all other functions in your project to ES6 arrow functions, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

You can do a search in Vscode for the "function" word, and correct as needed (I found 5):

![image](https://github.com/mailsg/medimate-frontend/assets/57421823/4eadcf5a-7d07-4402-9edd-d022f9329164)

Shouldn't take you more than 5-10 min so don't worry.  

That way, your code will meet the latest standards. 💪